cpp/src/grammar.hpp

```diff
@@ -33,15 +35,19 @@ class Grammar {
   // PEG parser
   peg::parser emerald_parser;
 
+public:
```

Comment:
Yo, could you please put this public stuff up with the other public code at the top of the class 💯🙏